contrib/vagrant/provision-util.sh

```diff
@@ -128,6 +128,9 @@ os::provision::set-os-env() {
   fi
   os::provision::set-bash-env "${origin_root}" "${config_root}" \
 "/root/${file_target}"
+
+  # Make symlinks to the bash completions for the openshift commands
+  ln -s ${origin_root}/contrib/completions/bash/* /etc/bash_completion.d/
```

Comment:
I'm new-ish to git, so my usage may be wrong.  However, I planned to merge
all changes after the reviewers said go, but before I committed.  Is that
reasonable?

-ben

On Mon, Dec 14, 2015 at 6:50 PM, Maru Newby <notifications@github.com>
wrote:

> In contrib/vagrant/provision-util.sh
> <https://github.com/openshift/origin/pull/6256#discussion_r47578031>:
>
> > @@ -128,6 +128,9 @@ os::provision::set-os-env() {
> >    fi
> >    os::provision::set-bash-env "${origin_root}" "${config_root}" \
> >  "/root/${file_target}"
> > +
> > +  # Make symlinks to the bash completions for the openshift commands
> > +  ln -s ${origin_root}/contrib/completions/bash/* /etc/bash_completion.d/
>
> In that case, lgtm.
>
> It seems strange to see you not rewriting your history, but it's saner in
> a way, too.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6256/files#r47578031>.
>